storeshare_api/db/migrate/20221030125338_create_renters.rb

```diff
@@ -0,0 +1,7 @@
+class CreateRenters < ActiveRecord::Migration[7.0]
+  def change
+    create_table :renters do |t|
+      t.timestamps
```

Comment:
Should 'renters' have a foreign key pointing to a user?

Reply:
Right now it works as a single table inheritance (only the user table gets generated with a 'type' field of either renter or subletter). I'll actually move away from this and make separate tables, so I'll need to add a foreign key when I do it

Reply:
sounds good!

Reply:
i'll leave the PR open til you get that change in before merging to main, since it's somewhat important to the schema

Reply:
done